tests/50federation/32room-getevent.pl

```diff
@@ -0,0 +1,49 @@
+test "Inbound federation can return events",
+   requires => [ $main::OUTBOUND_CLIENT, $main::HOMESERVER_INFO[0],
+                 local_user_and_room_fixtures() ],
+
+   do => sub {
+      my ( $outbound_client, $info, $user, $room_id ) = @_;
+      my $first_home_server = $info->server_name;
+
+      my $local_server_name = $outbound_client->server_name;
+
+      my $user_id = "\@50fed-user:$local_server_name";
```

Comment:
Hmm.. There's no great requirement for it to be the same though. I guess either make it a constant so it actually is, or dynamically allocate them so it's clear they're different. Probably easier to do the latter in fact. Would you mind if I do that on a separate PR?